app/views/browse/deployment.html

```diff
@@ -57,7 +57,8 @@
                         project-name="{{deployment.metadata.namespace}}"
                         alerts="alerts"
                         disable-delete="!!deployment.status.replicas"
-                        hpa-list="hpaForRC">
+                        hpa-list="hpaForRC"
+                        redirect-url="{{deployment | configURLForResource}}">
```

Comment:
@jwforres FYI, I tested this for vanilla RCs. Since `configURLForResource` returns null for replication controllers, it falls back to the default behavior.